extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -23,6 +23,26 @@ import {toggle} from '../../../src/style';
 
 
 class AmpStickyAd extends AMP.BaseElement {
+  /** @param {!AmpElement} element */
+  constructor(element) {
+    super(element);
+
+    /** @private {!Element} */
```

Comment:
no let's not change the signature of `scheduleLayout` but use `assertElement` on the call site if we can guarantee that you do have an element.